pkg-manager/core/test/install/fromRepo.ts

```diff
@@ -133,14 +134,14 @@ test('a subdependency is from a github repo with different name', async () => {
 
   const lockfile = await project.readLockfile()
   expect(lockfile.packages['/@pnpm.e2e/has-aliased-git-dependency@1.0.0'].dependencies).toStrictEqual({
-    '@pnpm.e2e/has-say-hi-peer': '1.0.0(hi@1.0.0)',
+    '@pnpm.e2e/has-say-hi-peer': '1.0.0(github.com/zkochan/hi/4cdebec76b7b9d1f6e219e06c42d92a6b8ea60cd)',
```

Comment:
oh awesome, this seems like it would fix #7393 wouldn't it?

Reply:
I am not sure. It was a byproduct of my changes. I'll be happy if it fixes more issues.